internal/util/update/update.go

```diff
@@ -142,17 +146,17 @@ func (u Command) Run() error {
 		u.Output = os.Stdout
 	}
 
-	kptfile, err := kptfileutil.ReadFileStrict(u.FullPackagePath)
+	orgKf, err := kptfileutil.ReadFileStrict(u.FullPackagePath)
```

Comment:
What does the name org signify here ? Original ?

Reply:
Yeah, org means original. I have updated the name of the variable to `rootKf` which I think is a better name.